app/lib/three_scale/redis_config.rb

```diff
@@ -23,11 +34,7 @@ def reverse_merge!(other)
     end
 
     def method_missing(method_sym, *args, &block)
-      if config.respond_to?(method_sym)
-        config.send(method_sym, *args, &block)
-      else
-        super
```

Comment:
This super was leading to nowhere, it seems.

Reply:
super should raise `NoMethodError` :)